python/ray/train/tests/test_logging.py

```diff
@@ -0,0 +1,173 @@
+import logging
+import os
+import re
+from datetime import datetime
+
+import pytest
+import yaml
+
+import ray
+from ray.tests.conftest import *  # noqa
```

Comment:
nit: don't think this is needed actually. conftest fixtures should automatically be visible

Reply:
Good point! I can remove this unnecessary import.